x-pack/plugins/cases/public/types.ts

```diff
@@ -51,7 +51,19 @@ export type StartServices = CoreStart &
     security: SecurityPluginSetup;
   };
 
+type CaseOwnerTypes = 'securitySolutionCases' | 'observabilityCases';
+export interface CasePermissionOptions {
```

Comment:
In this file there are only plugin top-level types, I don't think it is the proper place for this one, it would be better in the _methods/can_use_cases.ts_ itself, similar to what the other files in /methods are doing.

Reply:
I think we should stick to using `Cases` in the plural, unless it is referring to a specific case entity.

Reply:
makes sense. Thank you